src/cryptoadvance/specter/util/wallet_importer.py

```diff
@@ -266,7 +266,16 @@ def parse_wallet_data_import(cls, wallet_data):
                 xpubs += "[{}]{}/0/*,".format(
                     d["derivation"].replace("m", d["root_fingerprint"]), d["xpub"]
                 )
-                cosigners_types.append({"type": d["hw_type"], "label": d["label"]})
+                if "hw_type" in d:
+                    cosigners_types.append({"type": d["hw_type"], "label": d["label"]})
+                else:  # this can occcur if no hardware wallet was used, but the seed is available
+                    cosigners_types.append(
+                        {"type": "electrum", "label": f"Electrum Multisig {i}"}
+                    )
+                    if "seed" in d:
+                        logger.warning(
+                            "The Electrum wallet contains a seed. The seed will not be imported."
```

Comment:
This warning is not visible for the user. However I think it is important that a user is warned. How can I show this warning to the user?

Reply:
```suggestion
                        flash(_("The Electrum wallet contains a seed. The seed will not be imported."), "warning")
                            
```
This should work!

Reply:
Great. Thanks, I did this and added the missing imports.